resolver_test.go

```diff
@@ -0,0 +1,30 @@
+package doh
+
+import (
+	"context"
+	"testing"
+)
+
+func TestLookupIPAddr(t *testing.T) {
```

Comment:
Would be good to have an explicit check that confirms this returns both ipv4 AND ipv6.

Reply:
sure, let me do that/